src/scss/templates/_blog.scss

```diff
@@ -160,3 +178,32 @@
 	}
 
 }
+
+.blog-back-to-top {
+	position: fixed;
+	bottom: calc-em(20);
+    right: calc-em(15);
+    cursor: pointer;
+    z-index: $zindex-dropdown;
+	opacity: 0;
+	transition: 0.4s ease-in;
+	transition-property: opacity, bottom;
+	display: none;
+
+	&:hover {
+		color: $primary;
+	}
+
+	body.btt-btn--show & {
```

Comment:
how are you going to add that class to the body?

Reply:
> how are you going to add that class to the body?

I just reused the existing scrolling function. On scroll, this class is added.